### ISSUE TITLE
Better guidance on Base IRI

### ISSUE THREAD

**Issue by isakgustavsen** [andybywire/sanity-plugin-taxonomy-manager#6]
The current guidance on the Base IRI field is hard to follow. Is it possible to add some more context or examples to help?

**Comment by andybywire** [comment]
Thanks for filing this issue. I've added some additional guidance on Base IRIs (and Preferred Labels) to the README for this plugin, and have a to-do to make the in context descriptions easier to follow as part of some wider plugin updates and improvements planned for Q2 2023.
